apps/web/components/booking/BookingListItem.tsx

```diff
@@ -392,6 +392,9 @@ function BookingListItem(booking: BookingItemProps) {
               {isRejected && <div className="text-sm text-gray-500">{t("rejected")}</div>}
             </>
           ) : null}
+          {isPending ? (
```

Comment:
This can potentially cause duplicate Actions in UI.
**Theoretically:** For a booking that is in future and not accepted.
- `isUpcoming` is true
-` isCancelled` if false
- `isPending` is true

Reply:
Good point, I'll take a look and see what I can do to fix that.

Reply:
Added a "isPast" conditional to go along with "isPending". This should stop duplicate actions from occurring.

Reply:
Did some manual tests to make sure that the theoretical bug @hariombalhara brought up does not occur, this action only shows up on unconfirmed bookings in the past.